lib/private/AppFramework/Middleware/MiddlewareDispatcher.php

```diff
@@ -40,6 +40,7 @@
  */
 class MiddlewareDispatcher {
 	/**
+	 * @deprecated
```

Comment:
```suggestion
	 * @deprecated 26.0.0
```
And same for the other deprecated methods?

Reply:
it is internal. we don't need to spec a version

Reply:
I have reliazed that this prop will still be used. deprecation reverted.